torchvision/prototype/transforms/functional/_misc.py

```diff
@@ -15,10 +15,8 @@
 
 
 def normalize(inpt: DType, mean: List[float], std: List[float], inplace: bool = False) -> DType:
```

Comment:
Can we also have `Sequence[float]` here and convert to list again before calling the kernel? Since we already do this conversion in the transform, there will be no extra conversion when repeatedly calling this from there. On the other hand, users that only want the functional get the benefit of not only using lists.

Reply:
That's a bigger change to be discussed because you are changing the signature of a public method. `F.normalize()` exists already on the old API. Perhaps we can do this because we already change the type of inpt. But I wouldn't do this change right now.

Reply:
Let me add a bit more info here to have a bit more context.

If we had a way to detect that a certain Tensor object is a "simple tensor" while JIT-scripting, then we would be able refactor the entire `F` API to keep it JIT-scriptable and maintain BC. This could require some nasty tricks like:
- Faking types
- Splitting the non-jit scriptable part of `F` high-level kernels in separate methods that are marked with @torch.jit.unused. 
- Extensive use of `torch.jit.is_scripting()` and `torch.jit.is_tracing()`

Another thing we could do if we want to avoid BC breakages is to provide a jit-scriptable `F` that has exactly the same signatures as the high-level kernels but only supports pure tensors.

To keep the above a possibility, we shouldn't modify prematurely the existing signatures to receive parameters that are not JIT-scriptable (like `Sequence`). We might end up doing it at the end, but let's hold off to that until we know for sure.